### DISCUSSION TITLE
Relationship column as display column?

### DISCUSSION THREAD

**Discussion by kulamit** [Budibase/budibase#12707] [Help] [answered]
Hello All,

I want to display database table in grid format but I want to hide some of the columns from table such as row identifier of the table. How can I achieve it without using custom data repeater approach?
Column in table is of relationship type from another table and "Use as display Column" is disabled for it. Similarly row identifier column can't be hidden. Which makes it difficult to customize table view in UI design.
Similarly while creating view of table, can't hide/remove row identifier column.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @kulamit,

I agree this would be nice to do. Currently there isn't an option for this. I have gone ahead and created a feature request for this. Hopefully it will get picked up and implemented when the team have time. You can view this enhancement [here](https://github.com/Budibase/budibase/issues/12714).
